TakingITGlobal/commit2act
# [TIG-725] Remaining signup sub pages

## Ticket(s)

* [TIG-725](https://app.clickup.com/t/9009201449/TIG-725) (focused sub-task)
* [TIG-663](https://app.clickup.com/t/86drn3m14) (parent task)

## Description

This PR adds the remaining sub-page steps for the Sign Up flow - each with it's own story and test file. At this time, these placeholder components are nearly identical. As content and data handling is added to each, they'll grow more unique and complex. 

We're also exporting all sub-pages from a central `index` file, which allows us to import a full object from `PublicRoutes` and use the `SIGNUP_ROUTES` array to map and output the individual component per route. 

**Note:** The `InfoSecondary` page is shifted in this sequence to be the first onboarding step, after the user kicks-off the full onboarding flow. This is meant to group our user profile screens together, separately from the user preferences screens.

**Another Note:** You'll notice an `OnboardEnd` page here with the heading "You're all set!" that [isn't currently in the UI designs](https://www.figma.com/file/pLQlafaAJPCvcsLHGbO1Gh/Commit2Act-(Copy)?type=design&node-id=4667%3A5325&mode=design&t=U5KPPgRY7rfj0Sez-1). This is a placeholder for whatever success/congrats screen we will wind up using which is not yet designed.

## Testing

* Pull down and install the branch
* Run `npm run test` and confirm all tests pass
* Run `npm run storybook` and confirm all stories under Pages > Sign Up render with correct headings
* Run `npm run start` and visit `/account/create` once the app loads in browser
* Confirm that the Next/Back links navigate through each step and that the pages have their correct headings

## Screenshots

**Second step in the flow (`OnboardStart`) rendering with correct route:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/3f8c2c1b-3580-4fce-b6fd-e481e61a56fd)

**Final step (`OnboardEnd`) rendering with correct route and disabled "Next" link:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/0fc6d260-d2b5-4448-a0d5-2d61f9d77a3f)

**Storybook entries for all Sign Up pages, numbered in order:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/1e0262be-31fa-491f-916f-ce4fbdbba177)
